internal/transactionsubmission/services/channel_accounts_service.go

```diff
@@ -0,0 +1,377 @@
+package services
```

Comment:
I didn't thoroughly review the logic of each method here. I assumed the logic of each operation remained mostly unchanged.

Reply:
It did, yes. I just did some cleanup as I updated the method signatures to support the sig service being injected.